### DISCUSSION TITLE
今天启动器更新后，玩atm9 F3内存占用上下蹦迪 游戏也一顿一顿的，这是什么问题

### DISCUSSION THREAD

**Discussion by XIXIAQQ** [Meloong-Git/PCL#3842] [提问]
今天启动器更新后，玩atm9 F3内存占用上下蹦迪 游戏也一顿一顿的，这是什么问题

**Comment by wuliaodexiaoluo** [comment]
内存占用忽高忽低是因为 Java 的 GC 在工作（用于删除虚拟机内没有被引用的内存数据，从而释放空间）
卡顿的问题，如果是 oracle jdk，可以考虑使用 OpenJDK

**Comment by XIXIAQQ** [comment]
一样的有卡顿，但是昨天的情况就不是这样，只有内存满了才会出现卡顿，启动器更新之后游戏内存占用就出现了异常,从40-90来回循环每次从90—40的时候就会出现卡顿
